Source/Atomic/Resource/BackgroundLoader.cpp

```diff
@@ -264,20 +264,12 @@ void BackgroundLoader::FinishBackgroundLoading(BackgroundLoadItem& item)
     // If BeginLoad() phase was successful, call EndLoad() and get the final success/failure result
     if (success)
     {
-#ifdef ATOMIC_PROFILING
+#if ATOMIC_PROFILING
```

Comment:
This was reason why i did not add these comments elsewhere. We know that profiler subsystem was replaced therefore profiler changes should be ignored when merging from upstream. I am adding them to requested spots anyhow ;) Maybe we can find a way for got to help us do merges and make these comments thing of the past. I plan to look into it next ;)